README.md

```diff
@@ -44,6 +38,20 @@ Listening address where the raft APIs will be exposed.
 
 Listening port where the raft APIs will be exposed.
 
+    docker_admin_users:
+      - "{{ ansible_ssh_user }}"
```

Comment:
ansible_ssh_user is deprecated in favor to ansible_user regards to this: https://github.com/mitchellh/packer/issues/3275